### ISSUE TITLE
Suggestion: integrate this to Miko's repo via pull request

### ISSUE THREAD

**Issue by ynohat** [pmaugeri/ak-ehn-checker#1]
Hello Pascal,

Hope you're well!

I thought it was great that you should contribute a Dockerfile to @mikolajswider's script, but it feels more natural for the Dockerfile to live close to the script itself, which would also give him the opportunity to connect it to Dockerhub and make it available more simply for users, e.g. `docker run ak-ehn-checker`.

What do you think?

Best regards,
Anthony

**Comment by pmaugeri** [comment]
Yes totally agree. Please copy and paste this Dockerfile to Miko's repository. 
Then I will delete this repository that has no reason to live aside Miko's one.
